### ISSUE TITLE
Revamp

### ISSUE THREAD

**Issue by aslilac** [gleam-lang/website#198]
We've been talking a lot about the website the last few days, and I have some thoughts that I want to collect into this issue for initial discussion, rather than just jumping straight in and creating a dozen different ones.

Things to mention on the front page:
- .d.ts generation (https://twitter.com/JLarky/status/1593393956523417601)
- hex.pm
- available libraries (Nakai, Lustre, gleam_json, parser combinator, [rad](https://github.com/tynanbe/rad)
- Erlang/Elixir FFI (earmark, all the various Erlang HTTP libraries)

Things to use to build it:
- Nakai (suggested by Louis, not just me being pushy :p)
- Probably just Github actions + pages
- for CSS: keep it plain/use SCSS/use Tailwind?

**Comment by lpil** [comment]
Love it!

While I think long-term I would like to use Gleam + Nakai for the site I think it's best to do this incrementally and do a future rewrite separately to adding new content and restyling the site.

RE libraries, I want to make a dedicated site for discovering Gleam Hex packages. I'm weary of putting any specific ones on the main site, I think core should be equally supportive of all members of the ecosystem and if we (for example) gave our blessing to one library it'd be really hard for an alternative to get the recognision they deserve.

**Comment by inoas** [comment]
We could link the awesomegleam repo / readme?

**Comment by aslilac** [comment]
Yeah, linking to that or a site/separate page that shows some Gleam packages would also work. Just wanna have something to show "there are things around that you can use!" :)

**Comment by inoas** [comment]
> .d.ts generation (https://twitter.com/JLarky/status/1593393956523417601)

We could show off all 4 integration targets:
- Erlang + Elixir
- JavaScript + TypeScript

**Comment by lpil** [comment]
[Gleam.fig.zip](https://github.com/gleam-lang/website/files/10070857/Gleam.fig.zip)

@aslilac 's design magic
